File: VB/WindowsFormsApplication19/Form1.vb
Imports Microsoft.VisualBasic
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.ViewInfo

Namespace WindowsFormsApplication19
	Partial Public Class Form1
		Inherits Form
		Private helper As DragDropHelper

		Public Sub New()
			InitializeComponent()
			helper = New DragDropHelper(galleryControl1, galleryControl2)
			helper.EnableDragDrop()
		End Sub

		Protected Overrides Sub OnFormClosing(ByVal e As FormClosingEventArgs)
			MyBase.OnFormClosing(e)
			helper.DisableDragDrop()
		End Sub
	End Class
End Namespace

File: VB/WindowsFormsApplication19/Form1.cs
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.ViewInfo;

namespace WindowsFormsApplication19
{
	public partial class Form1 : Form
	{
        DragDropHelper helper;

		public Form1()
		{
			InitializeComponent();
            helper = new DragDropHelper(galleryControl1, galleryControl2);
            helper.EnableDragDrop();
		}

        protected override void OnFormClosing(FormClosingEventArgs e)
        {
            base.OnFormClosing(e);
            helper.DisableDragDrop();
        }
	}
}